package.xml

```diff
@@ -38,7 +38,7 @@
   <license>New BSD</license>
   <url type="website">http://ceres-solver.org/</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Ah, I see.  Because of the latter reason, I think it would be a better idea to just remove the dependency completely.  I'll do that.  Thanks for the explanation!